### PR TITLE
Implement TextEntryInteraction #10

### PR DESCRIPTION
**Pull request by ragingwind**
#10

### CODE REVIEW

File: packages/react/src/interactions/PlaceHolder.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+type PlaceHolderProps = TextEntryInteractionProps & ExtendedTextInteractionProps;
+
+export const getPlaceHolder = (props: PlaceHolderProps) => {
+  const {placeholderText, expectedLength, expectedLines} = props;
+  const expectedLengthText = expectedLength ? ` ${expectedLength} length expected` : '';
+  const expectedLineText = expectedLines ? ` ${expectedLines} line expected` : '';
+
+  return `${placeholderText || ''}${expectedLengthText}${expectedLineText}`;
+};
```
Comment (Il9) [inline packages/react/src/interactions/PlaceHolder.ts:14]: I think this is util

Comment (ragingwind) [reply packages/react/src/interactions/PlaceHolder.ts:14]: I might be but `getPlaceHolder` will be used in interaction not global. give me better solution

Comment (ragingwind) [reply packages/react/src/interactions/PlaceHolder.ts:14]: how about that it would be placed at `/utils/interactions.ts`?

Comment (Il9) [reply packages/react/src/interactions/PlaceHolder.ts:14]: good

**Pull request merged**